subworkflows/local/snv_annotation.nf

```diff
@@ -0,0 +1,74 @@
+// TODO: BCFTOOLS processes should have unique names so that they are not used multiple times in other workflows?
+include { ECHTVAR_ANNO                                  } from '../../modules/local/echtvar/anno/main'
+include { ECHTVAR_ENCODE                                } from '../../modules/local/echtvar/encode/main'
+include { BCFTOOLS_NORM                                 } from '../../modules/nf-core/bcftools/norm/main'
+include { BCFTOOLS_NORM as BCFTOOLS_NORM_SINGLESAMPLE   } from '../../modules/nf-core/bcftools/norm/main'
+include { BCFTOOLS_INDEX                                } from '../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_SINGLESAMPLE } from '../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_FILLTAGS                             } from '../../modules/local/bcftools/filltags/main'
+include { BCFTOOLS_FILLTAGS as BCFTOOLS_FILLTAGS_ANNO   } from '../../modules/local/bcftools/filltags/main'
+include { ENSEMBLVEP_VEP                                } from '../../modules/nf-core/ensemblvep/vep/main'
+
+workflow SNV_ANNOTATION {
+
+    take:
+    ch_bcf
+    ch_single_sample_vcf
+    ch_databases
+    ch_fasta
+    ch_vep_cache
+    val_vep_cache_version // string: [mandatory] default: 110
+
+    main:
+    ch_versions       = Channel.empty()
+
+    // Add allele count tag to mutlisample vcf
+    BCFTOOLS_FILLTAGS(ch_bcf)
+    // Index and normalize multisample vcf
+    BCFTOOLS_INDEX(BCFTOOLS_FILLTAGS.out.vcf)
+    BCFTOOLS_NORM(BCFTOOLS_FILLTAGS.out.vcf.join(BCFTOOLS_INDEX.out.csi), ch_fasta)
+
+    // Index and normalize single sample vcfs
+    BCFTOOLS_INDEX_SINGLESAMPLE(ch_single_sample_vcf)
+    ss = ch_single_sample_vcf.join(BCFTOOLS_INDEX_SINGLESAMPLE.out.csi)
```

Comment:
guessing that `ss` is "single sample" but it would be good to have a more explicit channel name.